Goal: Reload cyrus when cyrus.conf is changed

Related to #579933: kolabd: I can't enable/disable pop3/s and imap/s

Author: Mathieu Parent <math.parent@gmail.com>

Upstream status: Better change already commited to HEAD

Index: libkolab-perl-2.2.4-20100624/lib/Kolab/Conf.pm
===================================================================
--- libkolab-perl-2.2.4-20100624.orig/lib/Kolab/Conf.pm	2010-07-06 00:04:50.000000000 +0200
+++ libkolab-perl-2.2.4-20100624/lib/Kolab/Conf.pm	2010-07-06 00:05:08.000000000 +0200
@@ -297,7 +297,7 @@
                 $haschanged{'apache'} = 1;
             } elsif ($cfg =~ /openldap/) {
                 $haschanged{'slapd'} = 1;
-            } elsif ($cfg =~ /imapd/) {
+            } elsif ($cfg =~ /(imapd|cyrus)/) {
                 $haschanged{'imapd'} = 1;
 	    } elsif ($cfg =~ /amavisd/) {
                 $haschanged{'amavisd'} = 1;
